theme/src/css/_zendesk-article.scss

```diff
@@ -1,3 +1,9 @@
+#docsArticle .article-body {
```

Comment:
Do we need the id `#docsArticle` or is there a CSS class that we can leverage instead?  If not, it is good to go.

Thanks @rdai10!

Reply:
Yeah the `#docsArticle` id is intended to scope the table style changes to imported official docs. The docs team changed their parser so it's been looking kinda funky. I think that team is still working on it... I'm not sure what the long term resolution is yet, but this is a stopgap solution to make the tables look more readable for the time being. :(

Reply:
Got it.  Thanks for the clarification!